Library/Homebrew/extend/os/mac/keg.rb

```diff
@@ -3,4 +3,6 @@
 class Keg
   GENERIC_KEG_LINK_DIRECTORIES = (remove_const :KEG_LINK_DIRECTORIES).freeze
   KEG_LINK_DIRECTORIES = (GENERIC_KEG_LINK_DIRECTORIES + ["Frameworks"]).freeze
+  GENERIC_MUST_EXIST_DIRECTORIES = (remove_const :MUST_EXIST_DIRECTORIES).freeze
+  MUST_EXIST_DIRECTORIES = (GENERIC_MUST_EXIST_DIRECTORIES + [HOMEBREW_PREFIX/"Frameworks"]).uniq.sort.freeze
```

Comment:
```suggestion
  MUST_EXIST_DIRECTORIES = (GENERIC_MUST_EXIST_DIRECTORIES + [HOMEBREW_PREFIX/"Frameworks"]).sort.uniq.freeze
```

Reply:
I repeated this change in two more places in keg.rb, since I had copied this code from there: 20e85ac

this change implicitly made sense to me because I am familiar with the unix command-line tool `uniq`, but now that I look at the Ruby docs for `uniq`, I think the order does not matter to `uniq`, so I think this change is not actually necessary